File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcRelationship/IfcRelDecomposes/IfcRelAggregates/IfcRelAggregates.cs
public class IfcRelAggregates : IfcRelDecomposes
{
    public IfcObjectDefinition RelatingObject { get; set; }
    public SET [1:?] OF IfcObjectDefinition RelatedObjects { get; set; }

    // === WHERE CLAUSES ===
    // NoSelfReference : SIZEOF(QUERY(Temp <* RelatedObjects | RelatingObject :=: Temp)) = 0
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRoot/IfcRelationship/IfcRelDecomposes/IfcRelAggregates/IfcRelAggregates.vb
Public Class IfcRelAggregates Inherits IfcRelDecomposes
    Public RelatingObject As IfcObjectDefinition
    Public RelatedObjects As SET [1:?] OF IfcObjectDefinition

    ' === WHERE CLAUSES ===
    ' NoSelfReference : SIZEOF(QUERY(Temp <* RelatedObjects | RelatingObject :=: Temp)) = 0
End Class
